src/components/PlayField/PlayField.tsx

```diff
@@ -1,82 +1,54 @@
-import styles from "./PlayField.module.css";
-import Card from "../Card/Card";
 import React from "react";
-import { initialImages } from "../../constants/initialImages";
-import { IOpenCardsItem, IPlayingFieldItem } from "../../types/types";
-import { shuffleArray } from "../../utils/array";
+import Card from "../Card/Card";
+import { CardId } from "../../types/types";
+import styles from "./PlayField.module.css";
+import { getCardsById } from "./helpers";
 
-const images = shuffleArray([...initialImages, ...initialImages]);
-const initialCards: IPlayingFieldItem[] = images.map((url) => {
-  return {
-    url,
-    flip: false,
-    off: false,
-  };
-});
+const cardsById = getCardsById();
 const MAX_OPEN_CARDS = 2;
 
 function PlayField() {
-  const [openCards, setOpenCards] = React.useState<IOpenCardsItem[]>([]);
-  const [cards, setCards] = React.useState<IPlayingFieldItem[]>(initialCards);
-  const areBothCardsOpen = openCards.length === MAX_OPEN_CARDS;
+  const [openCardIds, setOpenCardIds] = React.useState<CardId[]>([]);
+  const [switchedCardIds, setSwitchedCardIds] = React.useState<CardId[]>([]);
+
```

Comment:
If you push the idea further, you can see that having `id` allows you to only track open and "off'ed" card ids